packages/medusa/src/api/routes/admin/inventory-items/list-location-levels.ts

```diff
@@ -0,0 +1,89 @@
+import { Request, Response } from "express"
+
+import { IInventoryService } from "../../../../interfaces"
+import { FindParams } from "../../../../types/common"
+
+/**
+ * @oas [get] /inventory-items/{id}/location-levels
+ * operationId: "GetInventoryItemsInventoryItemLocationLevels"
+ * summary: "List stock levels of a given location."
+ * description: "Retrives stock levels of a given location."
```

Comment:
Question: Should a location_id be optionally passed as part of the query?

The code here lists levels for an inventory item rather than for a location.

Reply:
sure, just thought I would ask to clarify, The only reason would be to get location levels for a subset of locations. I dont see it as being necessary 😄

Reply:
Ahh makes sense - missed the thinking that it could be an array i.e. `location_id: string[]` - that is a pretty good idea.

Reply:
In that case it makes sense to split this endpoint in two. list-inventory-item-levels and list-location-levels..
I can add that in another PR, does that make sense?

Reply:
Fine to leave for now